src/css/profile/mobile/common/popup.less

```diff
@@ -144,13 +149,7 @@
 	}
 
 	.ui-listview {
-		border-top-width: 1 * @px_base;
-		border-top-color: #e6e6e6;
-		border-top-style: solid;
-		border-bottom-width: 1 * @px_base;
-		border-bottom-color: #e6e6e6;
-		border-bottom-style: solid;
-		margin-left: -16 * @px_base;
+		margin-left: -23 * @px_base;
```

Comment:
Is there any reason to set the margin left with - 23px?

Reply:
Dialog itself has padding on the left side. 
The listview inserted inside also has a margin on the left, so it looks skewed to the right.
so, to adjust the spacing, type a value of -23px.

Reply:
Is the 24px of the dialog padding?
But why do you move to the left with -23px?

Reply:
When the header font was roboto-regular, 23px was an appropriate value.
But for roboto-medium, 24px is appropriate.
I will change it.